tests/suite/visualize/line.typ

```diff
@@ -74,8 +74,8 @@
 #line(length: 60pt, stroke: (paint: red, thickness: 1pt, dash: (1pt, 3pt, 9pt)))
 
 --- line-stroke-field-typo ---
-// Error: 29-56 unexpected key "thicknes", valid keys are "paint", "thickness", "cap", "join", "dash", and "miter-limit"
-#line(length: 60pt, stroke: (paint: red, thicknes: 1pt))
+// Error: 29-58 unexpected key "thickness_", valid keys are "paint", "thickness", "cap", "join", "dash", and "miter-limit"
```

Comment:
I think this was okay before because it is intended to be a typo

Reply:
I tend to propose this form (trailing `_`) because it shows "we're trying to make an accidental thing which is not really the right word" while still making it easy for spell-checkers (and there are many) not to complain. Otherwise each person who runs across has to build a list of "oh, that typo is intentional".